src/debug.c

```diff
@@ -526,8 +526,8 @@ void _serverAssertPrintClientInfo(client *c) {
 }
 
 void serverLogObjectDebugInfo(robj *o) {
-    serverLog(LL_WARNING,"Object type: %d", o->type);
-    serverLog(LL_WARNING,"Object encoding: %d", o->encoding);
+    serverLog(LL_WARNING,"Object type: %u", o->type);
+    serverLog(LL_WARNING,"Object encoding: %u", o->encoding);
```

Comment:
This is correct. Can you rebase so we can merge?

Reply:
Done.